static/lib/tinymce/tinymce_editor.css

```
@@ -37,7 +37,7 @@ body.dark_theme .page-main-inner.custom-page-wrapper {
 .page-main-inner.custom-page-wrapper p,
 .page-main-inner.custom-page-wrapper ul li {
     font-size: 16px;
-    font-family: 'Amulya', Arial, Helvetica, sans-serif;
+    font-family: 'Inter', 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)